### DISCUSSION TITLE
Test Discussion!

### DISCUSSION THREAD

**Discussion by gopalchandak-dev** [Cyfrin/foundry-full-course-cu#910] [General]
Feel free to say Hello Here!

**Comment by gopalchandak-dev** [comment]
I'm receiving this error while compiling:

' ' '
TypeError: Index expression cannot be omitted.
  --> FundMe.sol:41:13:
   |
41 |             addressToAmountFunded[] = 0;
   |             ^^^^^^^^^^^^^^^^^^^^^^^
' ' '
Here is my code:
' ' 'solidity

       for(uint256 funderIndex = 0; funderIndex < funders.length; funderIndex++){
            address funder = funders[funderIndex];
            addressToAmountFunded[] = 0;
        }

' ' '

Can someone help me find out what the issue is?

**Reply by 0xusmanf** [reply]
You are missing the `key` here `addressToAmountFunded[] = 0;` while assigning value in your mapping, in this case, `funder` is the key. Try the below code.

```solidity
for(uint256 funderIndex = 0; funderIndex < funders.length; funderIndex++){
        address funder = funders[funderIndex];
        addressToAmountFunded[funder] = 0;
    }
```

**Reply by gopalchandak-dev** [reply]
Thank you!
